tools/travis-upload-to-gdrive.sh

```diff
@@ -1,35 +1,57 @@
 #!/bin/bash
-
-source tools/travis-helpers.sh
+# This script uploads small- and big-test results to Google Drive. It uses
+# Google service account credentials serialized (base64) into
+# GDRIVE_SERVICE_ACCOUNT_CREDENTIALS environment variable. The script creates a
+# directory structure under GDRIVE_PARENT_DIR directory (should be set to gdrive
+# id of a directory; 'root' by default). The actual results are stored in a
+# tar.gz archive.
 
 set -euo pipefail
 
+source tools/travis-helpers.sh
+
 echo "Uploading test results to google drive"
 export GOPATH="/tmp/go"
 go get github.com/prasmussen/gdrive
 
-echo "${GDRIVE_SERVICE_ACCOUNT_CREDENTIALS}" | base64 --decode > /tmp/serviceAccountCredentials
+echo "${GDRIVE_SERVICE_ACCOUNT_CREDENTIALS}" | \
+    base64 --decode > /tmp/serviceAccountCredentials
 
+# Calls gdrive executable with set credentials
 gdrive() {
-    /tmp/go/bin/gdrive --config /tmp --service-account serviceAccountCredentials "$@"
+    /tmp/go/bin/gdrive --config /tmp \
+                       --service-account serviceAccountCredentials "$@"
 }
 
+# Gets ID of a file located under a specific parent.
 gdrive_get_id() {
```

Comment:
I wanted some comments like this:
```
# Gets ID of a file located under a specific parent.
# Usage:
# gdrive_get_id "$PARENT_ID" "$NAME"
```
But it's not strictly required.

Reply:
should be obvious now by the first lines of the function